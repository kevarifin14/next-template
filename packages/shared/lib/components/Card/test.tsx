import { render } from '@testing-library/react';
import React from 'react';

import { Card } from '.';

describe('Card', () => {
  it('matches snapshot', () => {
    const { baseElement } = render(<Card>this is a card</Card>);
    expect(baseElement).toMatchSnapshot();
  });
});
