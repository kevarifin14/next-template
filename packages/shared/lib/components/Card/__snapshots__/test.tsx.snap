// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches snapshot 1`] = `
<body>
  <div>
    <div
      class="px-6 py-8 rounded-3xl shadow-md dark:shadow-none bg-light dark:bg-dark-light"
    >
      this is a card
    </div>
  </div>
</body>
`;
